.github/workflows/test-suite.yml

```diff
@@ -103,9 +103,9 @@ jobs:
         id: pods-cache
         with:
           path: 'apps/bare-expo/ios/Pods'
-          key: ${{ runner.os }}-bare-expo-pods-${{ hashFiles('apps/bare-expo/ios/Podfile.lock') }}
+          key: ${{ runner.os }}-bare-expo-pods-v2-${{ hashFiles('apps/bare-expo/ios/Podfile.lock') }}
```

Comment:
If you're sure it's cache issue, I would just wait for it to be invalidated instead of changing the keys 😉 However, I wonder why because the lock has changed after upgrade 🤔

Reply:
the root cause is `installer.pods_project` be null. according to [this](https://github.com/CocoaPods/CocoaPods/issues/8604#issuecomment-472075100), bare-expo enabled `incremental_installation` make it possibly happened. that's why i tried not to use old cache.

do you know how to invalidate cache? according the [GH document](https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy), we should wait 7 days.

btw GH actions is [abnormal now](https://www.githubstatus.com/incidents/7p1nnvkgh96y), waiting for they fixing it.

Reply:
can reproduce the error locally by `pod install` twice. (which `incremental_installation` takes effect)
so i reverted the cache key change and further check if `installer.pod_project` is not nil.

Reply:
Afaik there is no way to immediately invalidate the cache, but I meant that any change in the lock should make it work 🙂 Also, there is a limit of 5GB for caches per repository, which I believe we achieve pretty quickly so older caches are for sure deleted in a few days.